spec.emu

```diff
@@ -895,10 +895,10 @@
         1. Let _sourcesContentCount_ be the the number of elements in _sourcesContent_.
         1. Let _sourceUrlPrefix_ be *""*.
         1. If _sourceRoot_ ≠ *null*, then
-          1. If _sourceRoot_ contains the code point U+002F (SOLIDUS), then
-            1. Let _idx_ be the index of the last occurrence of U+002F (SOLIDUS) in _sourceRoot_.
-            1. Set _sourceUrlPrefix_ to the substring of _sourceRoot_ from 0 to _idx_ + 1.
+          1. If _sourceRoot_ ends with the code point U+002F (SOLIDUS), then
+            1. Set _sourceUrlPrefix_ to _sourceRoot_.
           1. Else, set _sourceUrlPrefix_ to the string-concatenation of _sourceRoot_ and *"/"*.
+        1. TODO: Strip everything after last `/` from _baseURL_'s string representation.
```

Comment:
This is already done when we use the base URL, as it's doing the equivalent of `new URL(source, baseURL)` which removed the last segment from `baseURL` (e.g. `new URL("./foo.js", "http://example.com/folder/bar.js")` is `http://example.com/folder/foo.js`).

Reply:
Removed.